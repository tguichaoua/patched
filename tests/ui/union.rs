use patched::Patch;

#[derive(Patch)]
union Foo {
    a: u64,
    b: bool,
}

fn main() {}
