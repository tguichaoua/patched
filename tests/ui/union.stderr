error: Patch does not support derive for unions
 --> tests/ui/union.rs:4:1
  |
4 | / union Foo {
5 | |     a: u64,
6 | |     b: bool,
7 | | }
  | |_^
